### ISSUE TITLE
Polish refresh widget styling

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop-client#310]
Per the UX review today, the following tasks remain to be done on the refresh widget:

- [ ] The size of the refresh icon should match the [Zeplin auth widget spec](https://app.zeplin.io/project/5c807ea562f734bd2756b243/screen/5c86c91ea1202f046e7d92f4): 
  * static icon should be `15.1 x 15 pixels`
  * active icon (not in Zeplin yet) should just match size of static icon
  * disabled icon should be `20 x 19 pixels`
- [ ] Opacity of icons should be `92%`
- [x] After resizing the icon, we should still have a click area that's slightly larger than the icon itself, to ensure that users don't have to target it precisely 
- [ ] ~When in active state, the icon should be animated (["rotating at a pleasingly elegant pace... nothing frenetic or sluggy"](https://github.com/freedomofpress/securedrop-ux/issues/17#issuecomment-472164510))~
- [x] We should use title case for the label: "Last Refresh"
- [ ] The icon will need to be updated to the final version; @ninavizz will provide it in  https://github.com/freedomofpress/securedrop-ux/issues/17

**Comment by creviera** [comment]
Moved active state refresh animation here: https://github.com/freedomofpress/securedrop-client/issues/343

**Comment by ninavizz** [comment]
Note: Above spec states `15.1 x 15 pixels`. In one of our chats last week, I flagged to @creviera that my Sketch app is doing some weird scaling things... and that no pixel value should have decimal increments, outside of character or line spacing attributes for text. I have since worked to correct all artifacts in Sketch to reflect that; and any that remain in Zeplin, are outliers. FYI.

TL;DR, no sizing dimensions should include tenths of pixels.
